tensorflow/python/eager/context.py

```diff
@@ -1243,7 +1243,12 @@ def invoking_op_callbacks(self, value):
     self._thread_local_data.invoking_op_callbacks = value
 
   def _initialize_physical_devices(self, reinitialize=False):
-    """Get local devices visible to the system."""
+    """Get local devices visible to the system.
+    
+    Args:
+      reinitialize: This flag will reinitialize self._physical_devices so that
```

Comment:
Please rephrase to: "If True, reinitializes self._physical_devices so that..."

Reply:
done